components/layout.tsx

```diff
@@ -58,24 +63,33 @@ const TopNav: React.FC = () => {
     <>
       <Navbar
         bg="secondary"
+        sticky={isMobile ? "top" : undefined}
```

Comment:
For some reason, this isn't being set reliably in my testing. isMobile is true, but the html element doesn't end up with the classname. Things start working if I manually add `sticky-top` to the classNames of the navbar element.

Reply:
There's an issue with reloading the page, even if it's < 7XX px it's not adding sticky in the classes. I had to handle that tough way with a few different 'times' to check that. Please take a look, maybe you still would have a better way. Your way with classNames has the same issue with reloading - or perhaps I didn't make it the same way as you do.

Reply:
Ahh great point -- it's only an issue on reload and things work on rerender. I think this is due to ssr. useMediaQuery returns false  during the server render, so the initial html isn't sticky. Things work when we set `sticky` in a useEffect, which ensures it always rerenders with the correct sticky.

```ts
  const isMobile = useMediaQuery("(max-width: 768px)")
  const [sticky, setSticky] = useState(false)
  useEffect(() => setSticky(isMobile), [isMobile])
```